docs/posts/flink-unified-stream-batch/index.qmd

```diff
@@ -0,0 +1,414 @@
+---
+title: "Stream-batch unification through Ibis lens"
+author: "Chloe"
+date: 2024-02-08
```

Comment:
update -- let's either do this Thursday (this week) or Monday -- I want to get the roadmap out, so maybe roadmap Thursday and this Monday? then again maybe it's better to capitalize on the 8.0 launch? then again maybe it's better to wait and re-ignite the streaming fire next week? idk

let's plan on this going out Monday tentatively, roadmap this Thursday